Gemfile

```diff
@@ -26,6 +26,8 @@ gem 'delayed_job_active_record'
 gem 'daemons'
 gem 'foreman'
 gem 'c2', github: '18f/c2-api-client-ruby'
+gem 'jquery-rails-cdn'
```

Comment:
I think in general using a CDN is a lot of overhead for a site with our kind of traffic (aka - low) but in the case of jquery, since it's so easy, I think it is worth it. https://www.sitepoint.com/7-reasons-to-use-a-cdn/

> jQuery is ubiquitous on the web. There’s a high probability that someone visiting your pages has already visited a site using the Google CDN. Therefore, the file has already been cached by your browser and won’t need to be downloaded again.